Acceptance-Tests.md

```diff
@@ -247,6 +247,67 @@ On CI, we run all the acceptance tests in production mode, so the screenshots in
 
 On the other hand, if the screenshot fails locally (in desktop environment), the screenshot `teachPage-diff.png` will be generated and stored inside a new folder `diff-snapshots` under `logged-out-user/dev-desktop-screenshots`. 
 
+### Adding Prod screenshots for Acceptance Tests
+On making changes that affect a user journey tested through the acceptance tests or introduce a new feature, a contributor needs to update/add screenshots supporting their changes.
+It has been noted that screenshots obtained by running the Acceptance tests locally using the prod flag do not match those captured by the CI. To overcome this, you can commit any image in the prod screenshot folder and follow these steps:
```

Comment:
Drop the first sentence, replace with: "These screenshots should be obtained from the CI (not local) run, so that the environment matches future runs." Update the second sentence to follow on from that.

In the second sentence, what do you mean by "commit any image in the prod screenshot folder"?